kolibri/core/assets/src/utils/appCapabilities.js

```diff
@@ -18,6 +18,23 @@ export const checkCapability = key => store.getters.isAppContext && appCapabilit
 // checks and exposing the functions.
 
 export default {
+  /**
+   * @returns fn -> Promise<{ value: (boolean | null) }>
+   * Returns a function that returns a Promise that resolves to something responding to
+   * `data.value` whether it succeeds or fails.
+   */
+  checkIsMetered() {
+    if (!checkCapability('check_is_metered')) {
+      return Promise.resolve({ data: { value: null } });
+    }
+
+    const urlFunction = urls['kolibri:kolibri.plugins.app:appcommands-check-is-metered'];
+    if (!urlFunction) {
+      logging.warn('Checking if the device is metered is not supported on this platform');
+      return Promise.reject();
```

Comment:
This would happen if the capability was registered, but the app plugin was not enabled. Either way, I feel like it would be better to just resolve to `null` again like above?

So could maybe look for the URL function first, then just check:
```
if (!urlFunction || !checkCapability('check_is_metered')) {
```